Makefile

```diff
@@ -28,7 +28,7 @@ ifeq ($(V),1)
 BUILDV = -v
 endif
 
-TAGS =
+TAGS ?= pkcs11
```

Comment:
👍

Reply:
Thank you for pointing the thumb at a left-over :sweat_smile: 
I actually did the inverse build constraints first before I changed the tag to `nopkcs11` :upside_down_face:

Reply:
Oh, I thought you did it for documentation purposes! But yes, maybe you're right, it's better to remove it! 😄